### PR TITLE
4.17 GA

### PR DESCRIPTION
**Pull request by kalexand-rh**
Similar to #78050.

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@adellape, will you PTAL?

File: _distro_map.yml [hunk 1]
```diff
@@ -164,7 +170,7 @@ openshift-dedicated:
     enterprise-3.11:
       name: '3'
       dir: dedicated/3
-    enterprise-4.16:
+    enterprise-4.17:
```
Comment (kalexand-rh) [inline _distro_map.yml:173]: @karanthshashank, will you please confirm that OSD and ROSA will switch to 4.17 concurrent with OCP GA?

File: .s2i/httpd-cfg/01-commercial.conf [hunk 2]
```diff
@@ -161,7 +161,7 @@ AddType text/vtt                            vtt
 
     # Redirects for "latest" version
     RewriteRule ^(container-platform|enterprise)/?$ /container-platform/latest [R=301]
-    RewriteRule ^(container-platform|enterprise)/latest/?(.*)$ /container-platform/4\.16/$2 [NE,R=301]
+    RewriteRule ^(container-platform|enterprise)/latest/?(.*)$ /container-platform/4\.17/$2 [NE,R=301]
```
Comment (kalexand-rh) [inline .s2i/httpd-cfg/01-commercial.conf:164]: @bscott-rh, do these look right to you?

File: _distro_map.yml (hunk 1 above)
Comment (karanthshashank) [reply _distro_map.yml:173]: @kalexand-rh ACK.

File: .s2i/httpd-cfg/01-commercial.conf (hunk 2 above)
Comment (bscott-rh) [reply .s2i/httpd-cfg/01-commercial.conf:164]: Yes, both redirects LGTM

File: index-commercial.html [hunk 3]
```diff
@@ -329,6 +332,7 @@ <h2>OpenShift Container Platform <font size="-1">(한국어 문서)</font></h2>
             <div class="btn-group">
               <button type="button" class="btn btn-primary dropdown-toggle" data-toggle="dropdown" aria-expanded="false">버전 선택<span class="caret"></span></button>
               <ul class="dropdown-menu" role="menu">
+                <li><a href="https://access.redhat.com/documentation/ko-kr/openshift_container_platform/4.16/"><i class="fa fa-arrow-circle-o-right"></i> 4.17</a></li>
```
Comment (adellape) [inline index-commercial.html:335]: Should these two be `4.17` in the URL?

**Review by adellape — APPROVED** [review]

**Pull request merged**